tests/test_import_export.py

```diff
@@ -17,6 +17,21 @@
 from tests.data import expected as expected_data, input as input_data
 
 
+def _normalize_xml(dirname):
+    """Removes whitespace from xml files in the given directory."""
+    for dname, _, files in os.walk(dirname):
+        for fname in files:
+            fpath = os.path.join(dname, fname)
+            if not fpath.endswith('.xml'):
+                continue
+            with open(fpath) as f:
+                s = f.read()
```

Comment:
`pp_xml` calls out to etree in the case where libxml2-utils isn't installed (my system). I installed them, but still couldn't reproduce the error that travis sees. Installing libxml2-dev and libxml2-utils on Travis doesn't fix the problem.

Given both systems are ubuntu, it's unclear what the problem is. Possibly different versions of libxml2? I'm on 2.9.2 (ubuntu 15.04) and Travis (ubuntu 12.04) is on 2.7.8. 

Google-fu doesn't seem to fix this. I could potentially do an os.walk and use an xml diffing library to see if those files are different structurally. Would that be fine?